### PR TITLE
Kronecker batch ops

### PR DESCRIPTION
**Pull request by izmailovpavel**
Batch support for kronecker operations.

### CODE REVIEW

File: t3f/kronecker.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import numpy as np
 import tensorflow as tf
 
-from t3f.tensor_train import TensorTrain
+from t3f import TensorTrain, TensorTrainBatch
```
Comment (Bihaqo) [inline t3f/kronecker.py]: Please, use 
from t3f.tensor_train import TensorTrain
from t3f.tensor_train import TensorTrainBatch

I have no idea if there is any real difference, but 1) style guides usually suggest to not to import several things in one line 2) TF uses full absolute paths in imports (e.g. https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/ops/array_ops.py)

File: t3f/kronecker_test.py [hunk 2]
```diff
@@ -1,30 +1,28 @@
 import numpy as np
 import tensorflow as tf
 
-from t3f import tensor_train
-from t3f import kronecker as kr
-from t3f import ops
-from t3f import initializers
-from t3f import variables
+import t3f
```
Comment (Bihaqo) [inline t3f/kronecker_test.py]: Please, do not import t3f anywhere in the library. Again, not sure why it's bad, but noone does it)

File: t3f/kronecker.py (hunk 1 above)
Comment (izmailovpavel) [inline t3f/kronecker.py]: Done

File: t3f/kronecker_test.py (hunk 2 above)
Comment (izmailovpavel) [inline t3f/kronecker_test.py]: Done

**Pull request merged**